images/permissions/apps/sugarfixpermissions

```diff
@@ -1,14 +1,21 @@
-#!/bin/bash
-cd /var/www/html
-chown -R 1000:1000 .
-cd sugar
-if [ -e ./bin/sugarcrm ]
+#!/usr/bin/env bash
+
+DIR=/var/www/html
+
+# Only change ownership on files that aren't currently owned by sugar:sugar
+find ${DIR} -noleaf \( \! -user sugar -o \! -group sugar \) -print0 | xargs -0 -r chown sugar:sugar
+
+# Only change the mode of files/directories that are not 664/775
+find ${DIR} -noleaf -type d \! -perm 775 -print0 2> /dev/null | xargs -0 -r chmod 775 2> /dev/null
```

Comment:
Probably for these 2 lines we want to apply it only to ./sugar?

Reply:
Yes sorry, I'm actually running in /var/www/html without the subfolder in my images and missed that conversion.

Reply:
I'll open a new PR to fix that shortly.

Reply:
Hi John, don't worry, I've already merged it on the dev branch and modified it.

Now I am running into another issue that prevents it from merging it to master though, related to elasticsearch 62 permissions issues which does not fail locally but it fails on the build automation for some reason...

As soon as I pass the hurdle you will see it in master

Cheers!